aws-sns-topic/src/main/java/software/amazon/sns/topic/BaseHandlerStd.java

```diff
@@ -21,6 +23,7 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
 
   public static final int TOPIC_NAME_MAX_LENGTH = 256;
   public static final String FIFO_TOPIC_EXTENSION = ".fifo";
+  private static final int DELAY_TIME_MILLI_SECS = 8000;
```

Comment:
Just curious, why did you use 8000?

Reply:
I tried 3s, 5s, 8s. Only 8s can pass in PDX.🥲 It may not work in other regions, but let's see...